tests/integration/test_calico_integration.py

```diff
@@ -181,3 +181,30 @@ async def test_bgp_service_ip_advertisement(ops_test, bird_charm, kubernetes):
         'global-bgp-peers': '[]'
     })
     await bird_app.destroy()
+
+
+async def test_rp_filter_conflict(ops_test):
+    unit_number = 0
+    retcode, stdout, stderr = await ops_test.run(
+        'juju', 'ssh', '-m', ops_test.model_full_name, f'calico/{unit_number}',
```

Comment:
Super small nit, but you can `juju ssh calico/leader` without keeping track of a `unit_number`. It won't affect this 999/1000 times, but in that one chance when `calico/0` doesn't exist for some reason, `calico/leader` will be there for you.

```suggestion
        'juju', 'ssh', '-m', ops_test.model_full_name, f'calico/leader',
```